apps/ubuntu/resources.qrc

```diff
@@ -0,0 +1,43 @@
+<RCC>
```

Comment:
The thing is, in ubuntu I have two kits in QtCreator: one for desktop, and one for the phone using click packages.

For the click packages, I need to have install steps that install things to / (need to talk the sdk guy to find out if that's supposed to stay like that). However, the desktop build is ran directly from the build directory. In order to have the main.cpp find the qml files, I would need to copy them to the build dir. I can't do that with the install step to $$OUT_PWD as I did before, because that would install the things into the wrong place when building a click package.

So one way to solve this is the resource file, as they can be found with qrc://, regardless of the install paths.

I hear rumors that using a qrc file is supposed to be faster at loading too because it doesn't need to open many files, reads contents from ram when needed etc. I've yet to see numbers that prove this though.

That said, I've figured this causes the issue that files from the image cache can't be found any more with this branch. Not sure yet why. I need to figure that and then I'll decide if I want resources for ubuntu or not. You are free to choose if you want that or not.